client/src/components/Cards/index.js

```diff
@@ -10,7 +10,17 @@ const Cards = ({ data }) => {
     return (
       <div className="cards-containter">
         <div style={{ width: 220 * data.value.length }}>
```

Comment:
You should add `px` or other unit you want.

Reply:
Why? As far as I know, if I provide the value as a number it automatically takes it as 'px'. no?

Reply:
Browser helps you to avoide this mistake. 
Check your code in different browsers or add manually rule without unit. It will not works. 
According to [spec](https://www.w3.org/TR/CSS21/syndata.html#length-units) value should be with unit.

Reply:
What you say is correct for writing it in usual CSS or inline styles of usual HTML. But here it is inline styles of React, which is a bit different as it is compiled afterward to a normal HTML/CSS/JS, where it should automatically add all the units and everything else if needed, no? I have tried it in different browsers and everywhere it works fine and I see 'px' added to my value in the final html (http://prntscr.com/nimzoc). Let me know if I am wrong.

Reply:
I prefer to write usual and clear CSS everywhere :)
If I want to write width in pixels I will add `px`. It is obviously for me and for everyone else.
This is only my recomendation.

Reply:
Well,
> You SHOULD add `px` or other units you want.
does not sound like a recommendation 😉 

I should say that all the time I have seen someone using inline styles in React (courses on Coursera, YouTube, projects at work) - everywhere people were rather using values without units as it is cleaner and shorter and working correctly (win-win no?) This is why I used to write it like this...

In this particular case, this is how it looks like 

**with units**: `<div style={{ '${width: 220 * data.value.length}px' }}>`
**without units**: `<div style={{ width: 220 * data.value.length }}>`

This is so much cleaner and readable without units, no?

It is like to write a simple arrow function and add a `return` statement to it even if it can be omitted: `const func = (a, b) => { return a + b }`, no?

Thanks for the comment anyway, now I understand what you wanted to say and I see your point! Just in the beginning was a bit confused by the formulation of your comment so decided to clarify what did you mean. 🙂